example/example-metadata.ttl

```diff
@@ -237,7 +237,7 @@
    ].
 <test-abraham> dsp-repo:hasGivenName "Stewart" . 
 <test-abraham> dsp-repo:hasFamilyName "Abraham" . 
-<test-abraham> dsp-repo:hasEmail <stewart.abraham@test.ch> .
+<test-abraham> dsp-repo:hasEmail "stewart.abraham@test.ch" .
```

Comment:
If you want to be super correct, then you can write the email as "mailto:stewart.abraham@test.ch"^^xsd:anyURI. The `mailto:` is actually an URI shema (https://tools.ietf.org/html/rfc6068), qualifying the string after `mailto:` as as an email address.

Reply:
is there any actual benefit to that, apart from general correctness? (considering that we're still on the "quick and dirty iteration")  
it would mean some more rules to enforce in the form tool, which isn't a lot of work to do, but it still needs doing.

Reply:
It is more about how you want to store it, than what the GUI would need to do. The validation will need to happen, in the second iteration, on the backend.

Reply:
in that case I think I will suppress my perfectionism and leave it as it is for now